src/heitech.zer0mqXt.core/Main/Zer0Mq.cs

```diff
@@ -8,11 +9,13 @@ public class Zer0Mq : IZer0MqBuilder
     {
         private bool _isSilent;
         private ILogger _logger;
-        private ISerializerAdapter _serializer;
+        private TimeSpan _timeOut;
         private bool _usePublisher;
+        private ISerializerAdapter _serializer;
 
         private Zer0Mq()
         {
+            _timeOut = TimeSpan.FromMilliseconds(5000); // 5 seconds
```

Comment:
We recently talked about comments.
Yet, I'd prefer to use `TimeSpan.FromMilliseconds(5)` instead 😉.

Reply:
I guess you meant timespan.fromsecondes then? Else it would obv be a bit small :D

Reply:
Also this is an example of a rather not helpful comment ;)

Reply:
Indeed - I meant `TimeSpan.FromSeconds` _:Jean-Luc Picard facepalm meme:_.

Reply:
done